### ISSUE TITLE
feat: Notification toggle switch

### ISSUE THREAD

**Issue by uppified** [thorito/custom_activity_recognition#2]
Thank you for creating such a helpful package. I have found it to be incredibly useful compared to others on pub.dev.

However, I would like to request a small enhancement. Could you please add an option to enable or disable the foreground notification that pops up while using this package? This feature would greatly improve flexibility for developers and enhance the user experience in certain scenarios.

Thank you for your efforts and for maintaining this package. I look forward to your response and future updates.

**Comment by thorito** [comment]
Starting with Android 8.0, foreground services must display a persistent notification to the user.

This is due to resource usage and battery optimization restrictions imposed by Google to improve the user experience. 
If a service does not display this notification, the system may automatically terminate it.

Sources:

- https://developer.android.com/guide/components/foreground-services
- https://developer.android.com/about/versions/oreo/background

Note: Although it would be possible to remove the push notification once the service has started, this violates Android practices.

**Comment by thorito** [comment]
Added parameter in Android to hide push notification (not recommended according to Google policies)

https://pub.dev/packages/custom_activity_recognition
version: 0.0.11

```dart
  CustomActivityRecognition.instance.startTracking(
    showNotification: false, // <-- this
    useTransitionRecognition: false,
    useActivityRecognition: true,
    detectionIntervalMillis: 10000,
    confidenceThreshold: 50,
  );
```

closed #2
